app/Orchid/Screens/PlatformScreen.php

```diff
@@ -41,18 +39,11 @@ public function query(): array
      */
     public function commandBar(): array
     {
+        $url = "http://ec2-3-145-124-177.us-east-2.compute.amazonaws.com/";
```

Comment:
Could we move this to an env? In order to change it in each environment

Reply:
UH SI, como no se me ocurrio!